picking_dispatch/stock.py

```diff
@@ -92,7 +92,7 @@ def do_partial(self, cr, uid, ids, partial_datas, context=None):
                 dispatch_id = unfinished_dispatch_ids[dispatch_id]
             maybe_finished_dispatches.setdefault(
                 dispatch_id, []).append(move.id)
-        for dispatch_id, move_ids in maybe_finished_dispatches.iteritems():
+        for dispatch_id, move_ids in maybe_finished_dispatches.iteritems(): # pylint: disable=W1620
```

Comment:
items() is optimized for python 3 to be a generator,
iteritems() is optimized for python 2 to be a generator,
in python 2 items() is a list.

So I rather prefer to use the generator approach in py2.

http://goo.gl/QTyrGQ